Sources/Share/Dialogs/AppInvite/AppInvite.DeliveryMethod.swift

```diff
@@ -34,6 +34,8 @@ public extension AppInvite {
       switch sdkDestinationRepresentation {
       case .facebook: self = .facebook
       case .messenger: self = .messenger
+      @unknown default:
```

Comment:
This makes me a little nervous, what do you think about an assertion failure instead so we crash in development but not for users?

Reply:
Done.